src/main/webapp/add_event_page/script.js

```diff
@@ -0,0 +1,22 @@
+let map;
+function initMap(){
+    map = new google.maps.Map(document.getElementById("map"),{
+        center: {lat:32.8811, lng:-117.2376},
+        zoom: 16,
+    });
+
+    var marker = new google.maps.Marker({
+        position: {lat:32.8811, lng:-117.2376},
+        map: map,
+        draggable: true,
+    });
+
+    google.maps.event.addListener(marker, 'dragend', function(marker){
+        var latLng = marker.latLng; 
+        currentLatitude = latLng.lat();
+        currentLongitude = latLng.lng();
+        jQ("#latitude").val(currentLatitude);
```

Comment:
Out of curiosity: What does jQ("#latitude") and jQ("longitude") do? Have you been able to do a console.log() to verify that retrieving the longitude and latitude actually works and returns the expected values?

Reply:
I found an online source about it and said "Use event listener google.maps.event.addListener(marker, dragend, function(marker) To listen for changes in the marker position." The current code would support a draggable marker because the new location could be tracked. I have not implemented the feature that the returned value(coordinate) would be submitted to the form.

Reply:
Sure! currentLatitude will get the marker's latitude location and currentLongitude will get the longitude information.